### DISCUSSION TITLE
Alexa setup

### DISCUSSION THREAD

**Discussion by KEV-github** [nimroddolev/chime_tts#46] [General]
Hi

I'm looking for detailed instructions on how to set up Chime to work with Alexa. There is a Yaml example but was unsure where the goes and  could not get it to work in services as I was confused as to where and what to put in the chime url section. Any help would be appreciated.

**Comment by nimroddolev** [comment]
Hi @KEV-github , I am not sure from your message which part is causing you issues, but essentially the YAML example in the documentation includes a sequence of two service calls:

1. A service call to `chime_tts.say_url` which generates a new mp3 file in your HA instance's public folder, based on the parameters included. The service returns the URL.
3. A call to `notify.alexa_media` with the URL path from the first service call.

Unfortunately I don't own any Alexa speakers so I can't test it on real hardware.
I suggest you post to the [Home Assistant forum post for the Chime TTS integration](https://community.home-assistant.io/t/chime-tts-play-audio-before-after-tts-audio-lag-free/578430), where hopefully another user with an Alexa speaker can offer some assistance.

**Reply by dbullendsl** [reply]
If you haven't solved this problem, see my last entry over in the issues area under the question about chime_tts.say_url not working

**Reply by nimroddolev** [reply]
Hey @KEV-github, I added a new `FFmpeg Arguments` parameter to the `chime_tts.say` and `chime_tts.say_url` services, allowing you to list the conversion parameters you want to use for an FFmpeg conversion on the generated audio.

I don't own any Alexa speakers, but you should be able to do a quick search on the Home Assistant forum to find the right FFmpeg arguments to convert the audio for the Alexa specifications.

I created [a discussion here](https://github.com/nimroddolev/chime_tts/discussions/59), and it would be great if you could please test it out on your Alexa speaker and let me know if it works for your use case.

I created a new beta version with the changes. Please follow these steps to install it:
1. Open the <a href="https://my.home-assistant.io/redirect/hacs_repository/?owner=nimroddolev&repository=chime_tts&category=integration" title="Chime TTS HACS repository" target="_blank">Chime TTS repository in HACS</a>
1. Click on the `⋮` button in the top right
1. Select the `↻ Redownload` option
1. Enable the `Show beta versions` option
1. Select version `v0.10.2-beta1` from the updated version drop-down list
